events/pull-nodeschool.js

```diff
@@ -1,16 +1,25 @@
 'use strict'
 
-const request = require('request'),
-  yml = require('./yaml-sync'),
-  nodeGeocoder = require('node-geocoder'),
-  geoOpts = {
-    apiKey: process.env.MAPS_TOKEN,
-    formatter: null
-  },
-  geocoder = nodeGeocoder('google', 'https', geoOpts)
-
-request('http://nodeschool.io/chapters/list.json', {json: true}, (e, resp, list) => {
-  if (e || resp.statusCode !== 200) throw (e || new Error('response not 200' + resp.statusCode))
+const nodeGeocoder = require('node-geocoder')
+const request = require('request')
+
+const yml = require('./yaml-sync')
+const pkg = require('../package')
+
+const geocoder = nodeGeocoder('google', 'https', {
+  apiKey: process.env.MAPS_TOKEN,
+  formatter: null
+})
+
+request({
+  headers: { 'user-agent': `${pkg.name}/${pkg.version}` },
+  url: 'http://nodeschool.io/chapters/list.json',
+  json: true
+}, (err, resp, list) => {
+  if (err || resp.statusCode !== 200) {
+    throw (err || new Error('response not 200' + resp.statusCode))
```

Comment:
Some nitpicking for you; the error message should probably have a space added after `200`.